pkg/version/version_test.go

```diff
@@ -114,7 +115,8 @@ func TestGetVersionsGithub(t *testing.T) {
 	t.Run("error on bad addr", func(t *testing.T) {
 		version, err := GetLatestReleaseGithub("http://a.super.non.existant.domain/")
 		assert.Equal(t, "", version)
-		assert.EqualError(t, err, "Get \"http://a.super.non.existant.domain/\": dial tcp: lookup a.super.non.existant.domain: no such host")
+		msg := fmt.Sprint(err)
+		assert.True(t, strings.Contains(msg, "no such host") || strings.Contains(msg, "i/o timeout"))
```

Comment:
It is probably fine to just assert.Error(). The content of the error isn't so much important, and isn't necessarily directly under our control.

Reply:
@wcs1only agree. I fixed it.